### PR TITLE
fixed classification issue; added gitignore

### PR DESCRIPTION
**Pull request by walid0925**
when finetuning, the `num_labels` was being inherited from the original pretraining configuration, so the logits shape was equivalent to the number of labels in the pretraining task

also added a .gitignore with basic settings to clean up some of the git cruft

### CODE REVIEW

File: chemberta/finetune/finetune.py [hunk 1]
```diff
@@ -131,6 +124,8 @@ def finetune_single_dataset(dataset_name, run_dir):
     dataset_type = get_dataset_info(dataset_name)["dataset_type"]
     if dataset_type == "classification":
         model_class = RobertaForSequenceClassification
+        config.num_labels = len(np.unique(train_labels))
```
Comment (walid0925) [inline chemberta/finetune/finetune.py:127]: this is the relevant change

Comment (ElanaPearl) [reply chemberta/finetune/finetune.py:127]: theoretically would be better to know ahead of time how many classes we have since this could be slow if we tried it on a big dataset but for now it does the job

**Review by ElanaPearl — APPROVED** [review]
the ratio of style : code changes is amusing
